scripts/RELEASE_PROCESS.md

```diff
@@ -17,7 +17,7 @@
 
 7. Check the package upgrades are correct by running `python -m aea.cli check-packages` and `python scripts/check_package_versions_in_docs.py`. Commit if satisfied.
 
```

Comment:
goes for this as well then. `aea check-packages` and `tox -e package-version-checks` I believe

Reply:
Yes all of them should be with `tox`

Reply:
I'll open a separate PR for this